routes/quizzes.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
```

Comment:
We're using Prettier which formats with single quotes — is your IDE working with Prettier correctly? The code should be auto-formatted on save

Reply:
After I update with the latest master with Vicky's latest changes It's working fine.